src/params/screening.jl

```diff
@@ -0,0 +1,35 @@
+struct ScreeningParams
+    begin_day::Float64
```

Comment:
naming: since it is a `Float64` it is better to name it something like `start_time`, days would be integers

Reply:
ok, i change